tests/blake2b.rs

```diff
@@ -42,21 +46,18 @@ mod test {
         for vector in test_vectors.iter() {
             test_num += 1;
             let input = hex::decode(&vector.input).unwrap();
-            assert_eq!(
-                hex::decode(vector.out.clone()).unwrap(),
-                variable_blake2b(&input, &vector.key, 64),
-            );
+            if let Some(test_vec) = variable_blake2b(&input, &vector.key, 64) {
```

Comment:
This seems like a bit error-prone. How about, we remove all the `key`-fields from `fixtures/blake2b.json` (and all the keyed test vectors). Then, the result must always be equal.